src/NzbDrone.Core/Indexers/Torznab/TorznabSettings.cs

```diff
@@ -46,10 +47,18 @@ public TorznabSettingsValidator()
         }
     }
 
-    public class TorznabSettings : NewznabSettings
+    public class TorznabSettings : NewznabSettings, ITorrentIndexerSettings
     {
         private static readonly TorznabSettingsValidator Validator = new TorznabSettingsValidator();
 
+        public TorznabSettings()
+        {
+            MinimumSeeders = 1;
+        }
+
+        [FieldDefinition(5, Type = FieldType.Textbox, Label = "Minimum Seeders", HelpText = "Minimum number of seeders required.", Advanced = true)]
```

Comment:
I suggest we add a comment to NewznabSettings that Field 5 is in use, to avoid conflicts in the future.